packages/storybook/stories/Dialpad/Dialpad.stories.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Dialpad as DialpadComponent, useTheme } from '@azure/communication-react';
+import { mergeStyles } from '@fluentui/react';
+import { Canvas, Description, Heading, Props, Title } from '@storybook/addon-docs';
+import { Meta } from '@storybook/react/types-6-0';
+import React from 'react';
+import { COMPONENT_FOLDER_PREFIX } from '../constants';
+import { hiddenControl } from '../controlsUtils';
+import { CustomDialpadExample } from './snippets/CustomDialpad.snippet';
+import { DialpadExample } from './snippets/Dialpad.snippet';
+
+const ExampleDialpadText = require('!!raw-loader!./snippets/Dialpad.snippet.tsx').default;
+const CustomDialpadText = require('!!raw-loader!./snippets/CustomDialpad.snippet.tsx').default;
+
+const getDocs: () => JSX.Element = () => {
+  /* eslint-disable react/no-unescaped-entities */
+  return (
+    <>
+      <Title>Dialpad</Title>
+      <Description>
```

Comment:
Add a beta banner to these stories so its reflected that this is not part of the stable release yet

Reply:
(or make the component internal, and put in the Internal folder)

Reply:
updated